build/android/sign_apk.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2019 The Brave Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+import sys
+import subprocess
+import tempfile
+
+if len(sys.argv) != 8:
+  print("Incorrect number of parameters")
+  print("Usage: " + sys.argv[0] + " <zipalign_path>, <apksigner_path>, <unsigned_apk_path>, <key_path>, <key_passwd>, <prvt_key_passwd>, <key_name>")
+  sys.exit(1)
+
+zipalign_path = sys.argv[1]
```

Comment:
this is ok if this is definitely temporary, but normally you should use `argparse` for the arguments

Reply:
Changed that